src/test/resources/myWords

```diff
@@ -0,0 +1,4 @@
+rabbit, lion.
```

Comment:
maybe give an extension to the file, so that we know what format its data is in (such as `.txt`)